test/views/behaviours/NonUkAddressViewBehaviours.scala

```diff
@@ -82,7 +82,7 @@ trait NonUkAddressViewBehaviours extends ViewBehaviours {
           "show an error summary" in {
 
             val doc = asDocument(createView(form.withError(FormError(field._1, "error"))))
-            assertRenderedById(doc, "error-summary-title")
+            assertRenderedByClass(doc, "error-summary-title")
```

Comment:
Change "error-summary-title" to "govuk-error-summary"

Reply:
Done